terraform/ec2/main.tf

```diff
@@ -477,6 +478,17 @@ resource "null_resource" "ssm_validation" {
   }
 }
 
+resource "null_resource" "ssm_canary_metrics" {
+  depends_on = [null_resource.ssm_validation]
+  count      = !var.skip_validation && var.enable_ssm_validate && var.canary ? 1 : 0
+
+  provisioner "local-exec" {
+    command = <<-EOT
+        aws cloudwatch put-metric-data --metric-name Success --dimensions testcase=${local.testcase_name} --namespace "Otel/Canary" --value 1.0
```

Comment:
Maybe we can send more detail info like how long it takes for ssm installation? Though that logic can be put into the python script.

Reply:
SSM service has sent such metrics to owner of SSM package. This is only for canary test and will align with other canary testcases.